fitbenchmarking/results_processing/support_page.py

```diff
@@ -8,16 +8,15 @@
 import os
 
 from jinja2 import Environment, FileSystemLoader
-import numpy as np
 
 import fitbenchmarking
 from fitbenchmarking.results_processing import plots
```

Comment:
flake8 says that `fitbenchmarking.results_processing.plots` is imported but not used.  Can we piggyback on this PR to get rid of this line?

Reply:
Removed